bitrise.yml

```diff
@@ -7,6 +7,10 @@ app:
 
   - BITRISE_EMULATOR_NAME: android-25-armeabi-v7a
   - EMULATOR_PLATFORM: android-25
+  - BITRISE_EMULATOR_NAME19: android-19-armeabi-v7a
+  - EMULATOR_PLATFORM19: android-19
+  - BITRISE_EMULATOR_NAME24: android-24-armeabi-v7a
+  - EMULATOR_PLATFORM24: android-24
   - EMULATOR_TAG: google_apis
```

Comment:
Maybe it is good idea to test also `default` (non-google_apis) tag? At the time of writing for API 25 there is only `google_apis` system image but older ones have both.